### PR TITLE
IT-1668: manage firewall and docker control-network on amor nodes

### PR DESCRIPTION
**Pull request by adrienthebo**

### CODE REVIEW

File: role/amor.yaml [hunk 1]
```diff
@@ -3,6 +3,16 @@ classes:
   - "profile::core::common"
   - "docker"
   - "python"
+  - "docker::networks"
+
+docker::networks::networks:
```
Comment (adrienthebo) [inline role/amor.yaml]: I like the idea of managing the control network definition via Puppet - though the docker_network type/provider aren't amazing.

File: role/amor.yaml [hunk 2]
```diff
@@ -22,3 +32,4 @@ python::python_pips:
     ensure: "1.25.0"
 packages:
   - "git"
+firewall::ensure: "stopped"
```
Comment (adrienthebo) [inline role/amor.yaml]: We need to come up with a better way to share Hiera configuration with a generic docker profile. @luiscorralt @tribeiro this looks to be the final thing that we needed to route SAL traffic from auxtel nodes to amor.

File: role/amor.yaml [hunk 3]
```diff
@@ -3,6 +3,16 @@ classes:
   - "profile::core::common"
   - "docker"
   - "python"
+  - "docker::networks"
+
+docker::networks::networks:
+  control-network:
+    ensure: 'present'
+    driver: 'macvlan'
```
Comment (jhoblitt) [inline role/amor.yaml]: This seems reasonable to me with the exception that I don't know if any of the CSCs have previously been deployed with macvlan.

Comment (adrienthebo) [reply role/amor.yaml]: I can confirm that we've used `macvlan` - this configuration is live on ts-csc-generic-1 and auxtel-control-01.

**Pull request merged**